docs/standards/universal-profile/lsp6-key-manager.md

```diff
@@ -890,6 +890,22 @@ Since the Key Manager offers **relay execution** via signed message, it's import
 
 - Signed messages with sequential nonces should be **executed in order**, meaning a signed message with nonce 4 can't be executed before the signed message with nonce 3. This is a critical problem which can limit the usage of relay execution.
 
+:::warning
+
+When execution 3 sequential relay calls with sequential nonces in a single channel, you need to keep in mind the fact that if the first transaction does revert, the next 2 will revert as well. That happens because a nonce is assigned to each relay call.
+
+E.g.:
+
+- First relay call - nonce is 4
+- Second relay call - nonce is 5
+- Third relay call - nonce is 6
+
+Now there is also a nonce stored on chain in the LSP6 contract, one of the requirements for executing a relay call is to have the same nonce on chain as the one assigned to the executed relay call. After each successful execution the on chain nonce is incremented.
+
+Given that the on chian nonce is 4 and we are executing the relay calls from the example. The first relay call has matching nonces. If it is executed the on chain nonce will become 5 and now the second relay call has matching nonce with the on chain nonce, permiting exeution. Now if the first relay call reverts, the on chain nonce remains 4 and now it's not matching with the nonce assigned to the second relay call and so it cannot be executed.
```

Comment:
Suggest rewording:

```suggestion
Given the example above, the on-chain nonce is 4 and we are executing the relay calls.

**If the first relay call pass ✅**
- First relay call: nonce was 4 -> incremented to 5
- Second relay call: nonce was 5 -> incremented to 6
- Third relay call: nonce was 6 -> incremented to 7


**If the first relay call fails ❌ **
- **First relay call reverts ❌** nonce was 4 -> nonce remains 4
- Second relay call: nonce on-chain is 4 -> nonce used to sign was 5 = reverts ❌ with `InvalidRelayNonce`
- Third relay call: nonce on-chain is 5 -> nonce used to sign was 6 = reverts ❌ with `InvalidRelayNonce`
```

Reply:
Can even add the link to the custom error `InvalidRelayNonce` in the Key Manager contract ABI docs.